spot_rl_experiments/spot_rl/envs/place_env.py

```diff
@@ -219,4 +223,181 @@ def get_observations(self):
             "joint": self.get_arm_joints(self.config.SEMANTIC_PLACE_JOINT_BLACKLIST),
             "is_holding": np.ones((1,)),
         }
+        print("---")
+        return observations
+
+
+class SpotSemanticPlaceEEEnv(SpotBaseEnv):
```

Comment:
we can have this inherit from SpotSemanticPlaceEnv and reduce a lot of duplicate code.